README.md

```diff
@@ -11,11 +11,77 @@ Aside from **yt** itself, the following packages are required to use yt_geotiff:
 - [gdal](https://gdal.org/)
 - [rasterio](https://rasterio.readthedocs.io/en/latest/)
 
-## Capabilities
+## Developments and working examples
 
-Currently this package is still in its early development stage. Once this stage is fully implemented, the package will have the following capabilities:
-- Read _geotiff_ files into **yt** using the yt.load function
-- Manage multiband files
-- Enable the selection of sub-regions for analysis
+A key developments applied to the yt_geotiff package include the enabling of rasterio Window-reads. This Window-read feature allows for yt_geotiff users to read a sub-region of a multiband raster image analysis without having to read the entire image. Reading only an area of interest allows user to work more efficiently and circumvents the issue whereby the raster image size exceeds a computer's RAM. The function uses exisitng funcitonality in yt in order to define Window-read areas based on rectangular and circular yt data container shapes.
 
-The aim of this is to enable inspection and analysis of georeferenced data with **yt** to take advantage of its existing capabilities.
+Import yt and the yt_geotiff extensions
+```
+import yt
+import yt.extensions.geotiff
+```
+
+Users of yt_geotiff can load a raster image to first extract image metadata, including pixel dimensions and the coordinate reference system:
+```
+ds = yt.load(raster_file)
```

Comment:
I would go ahead and use the filename for the sample image we've been using. We're hosting that as sample data, so the user could grab it and try this out by copying exactly what's here.